cmd/controller/main.go

```diff
@@ -80,6 +80,7 @@ const (
 	kubeconfigFlag               = "kubeconfig"
 	allocationBatchWaitTime      = "allocation-batch-wait-time"
 	defaultResync                = 30 * time.Second
+	enableSplitControllerFlag    = "enable-split-controller"
```

Comment:
This doesn't need to be a flag - you have access to the feature flag via `runtime.FeatureEnabled`

Reply:
Changed to a `runtime.FeatureEnabled(runtime.FeatureSplitControllerAndExtensions)`